charms/promote-all-charms-and-bundles.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -eux
+
+CHARMS="
+  cs:~containers/easyrsa
+  cs:~containers/etcd
+  cs:~containers/flannel
+  cs:~containers/kubernetes-master
+  cs:~containers/kubernetes-worker
+  cs:~containers/kubeapi-load-balancer
+  cs:~containers/kubernetes-e2e
+  cs:~containers/kubernetes-core
+  cs:~containers/canonical-kubernetes
+"
+
+for charm in $CHARMS; do
+  CHARM="$charm" FROM_CHANNEL="$FROM_CHANNEL" TO_CHANNEL="$TO_CHANNEL" ./charms/promote-charm.sh
```

Comment:
We'd definitely need `FROM_CHANNEL` either way. Resource revisions can differ depending on which channel you get it from, even if the charm revision is the same.

So when we promote from candidate to stable, for example, we better make sure we're getting the resources from candidate.

I'm also trying to make sure we don't have to put in manual revision #s since we've had problems with that in the past. The automation can use them as needed (and promote-charm.sh does use them) but we gotta do our best to keep that internal, IMO.